### PR TITLE
Anton dev

### PR DESCRIPTION
**Pull request by Leonezz**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/anton/anton.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
+
```
Comment (drashna) [inline keyboards/anton/anton.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anton/anton.c [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+#include "anton.h"
+
+#define MEDIA_KEY_DELAY 10
```
Comment (drashna) [inline keyboards/anton/anton.c:4]:
```suggestion
```

File: keyboards/anton/anton.c [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        0x04, 0x04,       0x04,             0x04,                   0x04, 0x04, 0x04, 0x04, 0x04, 0x04,
+    }
+};
+#endif
+uint8_t layer = 0;
+
+uint32_t layer_state_set_kb(uint32_t state) {
+  state =    (state);
+  layer = biton32(state);
+  return state;
```
Comment (drashna) [inline keyboards/anton/anton.c:42]: This isn't needed, and should be removed. 
```suggestion
```

File: keyboards/anton/anton.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#pragma once
```
Comment (drashna) [inline keyboards/anton/anton.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/anton/config.h [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6324
+#define DEVICE_VER      0x0001
+#define MANUFACTURER ZHUWENQ THE BEST
+#define PRODUCT ANTON
+#define DESCRIPTION Anton ver-1.0
```
Comment (drashna) [inline keyboards/anton/config.h:26]:
```suggestion
```

File: keyboards/anton/rules.mk [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# MCU name
+MCU = STM32F103
+
+# Bootloader selection
+BOARD = STM32F103C8_BOARD_FOR_ANTON
```
Comment (drashna) [inline keyboards/anton/rules.mk:5]: Is this actually needed?  Most of the settings are set by default.

**Review by drashna — CHANGES_REQUESTED** [review]
Could you run `qmk chibios-confmigrate` for your keyboard?

File: keyboards/anton/anton.c (hunk 1 above)
Comment (Leonezz) [reply keyboards/anton/anton.c:16]: license header added in the latest commit.

File: keyboards/anton/anton.h (hunk 4 above)
Comment (Leonezz) [reply keyboards/anton/anton.h:17]: license header added in the latest commit.

File: keyboards/anton/rules.mk (hunk 6 above)
Comment (Leonezz) [reply keyboards/anton/rules.mk]: I tried the `STM32_F103_STM32DUINO` board and I got a build pass. So I changed it to `BOARD = STM32_F103_STM32DUINO` in the latest commit.

**Comment by Leonezz** [comment]
Thanks a lot for the review, I will close this pull request and try a new one.

**Pull request closed without merging**